StackExchange.Redis.StrongName/StackExchange.Redis.StrongName.csproj

```diff
@@ -21,27 +21,16 @@
   </ItemGroup>
 
   <PropertyGroup Condition=" '$(TargetFramework)' == 'net45' or '$(TargetFramework)' == 'net46'">
-    <DefineConstants>$(DefineConstants);FEATURE_SERIALIZATION;FEATURE_SOCKET_MODE_POLL;FEATURE_PERFCOUNTER;FEATURE_THREADPOOL</DefineConstants>
+    <DefineConstants>$(DefineConstants);FEATURE_SOCKET_MODE_POLL;FEATURE_PERFCOUNTER;FEATURE_THREADPOOL</DefineConstants>
```

Comment:
`FEATURE_THREADPOOL` no longer used

Reply:
Doh, indeed forgot to nuke these - follow-up commit incoming :)